store/monitor/bin/boot

```diff
@@ -6,10 +6,13 @@ set -e
 ETCD_PORT=${ETCD_PORT:-4001}
 ETCD="$HOST:$ETCD_PORT"
 ETCD_PATH=${ETCD_PATH:-/deis/store}
-NUM_STORES=${NUM_STORES:-3}
-PG_NUM=${PG_NUM:-128} # default for 3 OSDs
 HOSTNAME=`hostname`
 
+# These defaults are for 3 hosts
+NUM_STORES=${NUM_STORES:-3}
+PG_NUM=${PG_NUM:-64}
+PGS_PER_OSD_WARNING=${PGS_PER_OSD_WARNING:-800} # 12 pools * 64 pgs per pool = 768
```

Comment:
This should not be the current number (1536) to avoid the warning in an existing cluster?